### ISSUE TITLE
Minimized panels: support a way to show in previous line

### ISSUE THREAD

**Issue by damithc** [MarkBind/markbind#154]
Current: minimized panels appear in a new line, although multiple minimized panels can appear in a single line.
![image](https://user-images.githubusercontent.com/1673303/36472928-1d4a53de-172e-11e8-94de-2c88357c723f.png)

Suggest: support a way to push a panel that are minimized by default to the end of the previous line. 
Reason: Saves vertical space (more printer friendly)

Example:
> Some text. Some text. <kbd>Minimized Panel 1</kbd><kbd>Panel 2</kbd>

**Comment by damithc** [comment]
You mean it is like that already?

**Comment by damithc** [comment]
It works only for trivial cases. If you add blank lines inside one of the panels, it won't work anymore. This code results in totally unexpected behavior:
```html
Some text, some text
<panel type="info" header="Minimized panel 1" minimized >

* Content 1
* Content 2

</panel>
```

**Comment by yucheng11122017** [comment]
Hello! I just tried out the code and currently this gives: 
```
Some text, some text

<panel type="info" header="Minimized panel 1" minimized >

* Content 1
* Content 2 
</panel>
<panel type="info" header="Minimized panel 1" minimized >

* Content 1
* Content 2
</panel>
```

to give 
![image](https://github.com/MarkBind/markbind/assets/77204346/762b8f86-fe92-4983-9eb6-9d32e0a86d74). 
However, removing the new line:
```
Some text, some text
<panel type="info" header="Minimized panel 1" minimized >

* Content 1
* Content 2 
</panel>
<panel type="info" header="Minimized panel 1" minimized >

* Content 1
* Content 2
</panel>
``` 

results in 
![image](https://github.com/MarkBind/markbind/assets/77204346/a041355b-f713-4a42-80c9-30b38e1837c1)

Seems like its an issue with the new line and parsing..

**Comment by lhw-1** [comment]
For the original issue, I am unsure if there is still a need to show minimized panels on one line, given that users can always alternate towards popover components instead. And given that we have ways to [hide minimized panels for print view](https://markbind.org/userGuide/reusingContents.html#optimizing-the-print-view), introduced into the User Guide in 2019 (after discussion on this issue died down), I am wondering if the issue is still relevant @damithc?

On a separate note, @yucheng11122017 I think we can file the bug you found as a separate issue relating to newlines **if** we decide to close the original issue 👍

**Comment by yucheng11122017** [comment]
> On a separate note, @yucheng11122017 I think we can file the bug you found as a separate issue relating to newlines **if** we decide to close the original issue 👍

Yup I think this makes sense! We can include it in issue #500 if that makes sense

**Comment by damithc** [comment]
> For the original issue, I am unsure if there is still a need to show minimized panels on one line, given that users can always alternate towards popover components instead. And given that we have ways to [hide minimized panels for print view](https://markbind.org/userGuide/reusingContents.html#optimizing-the-print-view), introduced into the User Guide in 2019 (after discussion on this issue died down), I am wondering if the issue is still relevant @damithc?

Although it is not explicitly stated in my original description, what I'm aiming for is to use panels to achieve _inline expansion/collapse of content_ so that the reader can control the level of details in a contiguous chunk of content. Content that pops doesn't give the same visual outcome.  However, I'm fine to let panels be what they are now, and use an entirely different syntax for inline-expandable text. e.g., 

```
Bat is a nocturnal animal. <inline header="more info on their behavior">They
 hunt at night and sleep during the day</inline>
```
